feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/weld/main/module.xml

```diff
@@ -44,7 +44,7 @@
         <module name="org.jboss.as.weld.beanvalidation" optional="true" services="import" />
         <module name="org.jboss.as.weld.webservices" optional="true" services="import" />
         <module name="org.jboss.as.weld.transactions" optional="true" services="import" />
-        <module name="javax.api"/>
+	<module name="java.naming"/>
         <module name="javax.annotation.api"/>
         <module name="javax.enterprise.api"/>
         <module name="org.wildfly.cdi-api-bridge"/>
```

Comment:
Looks like there's a missing dependency on ``java.xml``.  Maybe it comes in indirectly via another dependency?